src/styles/HomeStyles.js

```diff
@@ -5,6 +5,29 @@ const useStyles = makeStyles((theme) => ({
     width: "100%",
     backgroundColor: theme.palette.background.paper,
   },
+  
+  h1: {
```

Comment:
I'd be careful naming a class h1 when it's already such a used term, materialUI already defines styles for h1's in general so it could get confusing having different h1 elements and h1 classes and stuff. Probably something specific like EventsHeading or something might be better for now and we can establish generic styles that all our components can inherit later.